### DISCUSSION TITLE
Insecure password research

### DISCUSSION THREAD

**Discussion by adrianrudnik** [dani-garcia/vaultwarden#3860] [Ideas]
Provide a way to find entries that are related to insecure passwords.

I'm talking about the basic use case that a password was lost because you know it. Not the ones coming from https://bitwarden.com/help/reports/#exposed-passwords-report.

**Comment by BlackDex** [comment]
I would suggest to ask this at Bitwarden, and maybe there already is something like this requested.
This is not something Vaultwarden will built, since this is something on the client side, which we do not develop on our side.

**Reply by adrianrudnik** [reply]
Ah thanks, was under the impressum `/#/reports` was customized, thank you for the hint!
